RecoLocalCalo/EcalRecProducers/python/ecalMultiFitUncalibRecHit_cfi.py

```diff
@@ -14,7 +14,8 @@
       activeBXs = cms.vint32(-5,-4,-3,-2,-1,0,1,2,3,4),
       ampErrorCalculation = cms.bool(True),
       useLumiInfoRunHeader = cms.bool(True),
-  
+
+      gainSwitchEBMaxSample = cms.bool(False),
```

Comment:
what's a good reason for using different names and a different set of switches for the same thing in 80X and 90X? ```gainSwitchEBMaxSample``` in this PR (80X) vs ```gainSwitchUseMaxSampleEB``` and ```gainSwitchUseMaxSampleEE``` in #17279 (90X)